repo/packages/C/confluent/3/marathon.json.mustache

```diff
@@ -0,0 +1,151 @@
+{
+  "id": "{{service.name}}",
+  "cpus": 1.0,
+  "mem": 1230,
+  "instances": 1,
+  "cmd": "export PATH=$PATH:$MESOS_SANDBOX/jre/bin && ./scheduler/bin/kafka-scheduler server ./scheduler/conf/scheduler.yml",

```

Comment:
With the latest JRE package (see comment in resources.json) we're using pure stock JRE, which puts the contents under a "jre-<version>" subdir. Therefore it'd make sense to modify this command to instead say

```
  "cmd": "export PATH=$(ls -d $MESOS_SANDBOX/jre*/bin):$PATH && ./scheduler/bin/kafka-scheduler server ./scheduler/conf/scheduler.yml",
```

(with asterisks in the "jre" names)
(see also: https://github.com/mesosphere/universe-1.7/blob/version-2.x/repo/packages/K/kafka/15/marathon.json.mustache#L6 )